fixtures/redirects_redirect.sk.json

```diff
@@ -1,3 +1,3 @@
 {
   "/sk/pripadove-studie/": "/sk/naco-infozakon/"
-}
+}
```

Comment:
Preco si zmazal EOL?

Edit: Hm, pozeram ze ten EOL maze samotny `JsonFile`. A pozeram, ze ta classa nie je na citanie JSON suborov, ale na editaciu. Tj. ze vo vnutri `with` vies `data` upravit. A na konci `with` sa upravene data savnu. To nam v nasom pripade netreba. Tak teda nepouzivaj `JsonFile` classu, ale v `load_redirects` proste ten subor loadni.

Reply:
:ok: